server/src/main/resources/routes

```diff
@@ -50,3 +50,6 @@ PUT /admin/tables/:tableName/:ID net.wiringbits.webapp.utils.admin.controllers.A
 
 # delete table resource
 DELETE /admin/tables/:tableName/:ID net.wiringbits.webapp.utils.admin.controllers.AdminController.delete(tableName, ID)
+
+# get configs
+GET /config controllers.ConfigController.getConfig()
```

Comment:
Let's change this to `GET /environment-config controllers.EnvironmentConfigController.getEnvironmentConfig`